src/index.tsx

```diff
@@ -53,6 +55,16 @@ const memoryOnlyPrecomputedFlagsStore: IConfigurationStore<PrecomputedFlag> =
 const memoryOnlyPrecomputedBanditsStore: IConfigurationStore<IObfuscatedPrecomputedBandit> =
   new MemoryOnlyConfigurationStore();
 
+/**
+ * Builds a storage key suffix from an API key.
+ * @param apiKey - The API key to build the suffix from
+ * @returns A string suffix for storage keys
+ */
+function buildStorageKeySuffix(apiKey: string): string {
+  // Note that we use the first 8 characters of the API key to create per-API key persistent storages and caches
+  return apiKey.replace(/\W/g, '').substring(0, 8);
+}
+
```

Comment:
Copied from https://github.com/Eppo-exp/js-client-sdk/blob/dced1c2ded84a2173e6394542b16dd700da6931e/src/index.ts#L266-L275

But I don't believe this method needs to be public, so I didn't do that